apps/web/src/styles.css

```
@@ -404,6 +404,98 @@ a {
+   "border") plus a blurred copy behind it (the "shadow") that spin once. The
+   spin is an animated @property angle; the delay lets the home-page entrance
+   animations finish first. Desktop-Chrome only, so modern CSS is safe. */
+@property --install-rainbow-angle {
+  syntax: '<angle>';
+  initial-value: 0deg;
+  inherits: false;
+}
+@keyframes install-rainbow-spin {
+  0% {
+    --install-rainbow-angle: 0deg;
+    opacity: 0;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)